views/index.ejs

```diff
@@ -0,0 +1,9 @@
+<div class="conatiner">
+  <div class="well">
+    <h1>Url Shortener!!!</h1>
+    <form action="/links" method="POST">
+      <input type="text" name="urlForm">
```

Comment:
to get a better front end check for HTTP:// you can make the input type:url rather than type:text